inference-engine/src/transformations/src/transformations/op_conversions/convert_gather_v7_to_gather_v1.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_gather_v7_to_gather_v1.hpp"
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset7.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+#include "itt.hpp"
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertGather7ToGather1, "ConvertGather7ToGather1", 0);
+
+ngraph::pass::ConvertGather7ToGather1::ConvertGather7ToGather1() {
+    MATCHER_SCOPE(ConvertGather7ToGather1);
+
+    auto gather_v7 = pattern::wrap_type<ngraph::opset7::Gather>();
+
+    ngraph::matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto gather_v7_node = std::dynamic_pointer_cast<ngraph::opset7::Gather>(m.get_match_root());
+        if (!gather_v7_node)
+            return false;
+
+        int64_t batch_dims = 0;
+        if (gather_v7_node->is_axis_set())
```

Comment:
Why do we care about axis here? The Gather7 can be converted to Gather1 iff `axis==0`. If the value of `axis` is not set and `batch_dims` is negative then the `batch_dims` variable will be negative and the `return false;` will be triggered. If the `batch_dims` is non-negative then it doesn't really matter what is the value of `axis`: the `if (batch_dims != 0)` check will return false when it is necessary. So, no need to check whether axis is set or not.

Please, add a couple of unit tests for these cases.